src/cascadia/TerminalApp/Resources/en-US/Resources.resw

```diff
@@ -823,4 +823,8 @@
   <data name="ClosePaneToolTip" xml:space="preserve">
     <value>Close the active pane if multiple panes are present</value>
   </data>
+  <data name="TabColorClearButton.[using:Windows.UI.Xaml.Automation]AutomationProperties.FullDescription" xml:space="preserve">
+    <value>Reset tab colour</value>

```

Comment:
```suggestion
    <value>Reset tab color</value>
```

Sorry, I know it's a nit - we generally keep to en-US in the codebase, mostly because it is a good base language for the localizers. I know that this would likely not cause an issue, but I'm nitpicking it for consistency. :smile:

Reply:
Not a nit at all, don't worry!, I forget being UK based!